utils/defaults/defaults_test.go

```diff
@@ -164,3 +164,10 @@ func TestGetExperimentProgressDeadlineSecondsOrDefault(t *testing.T) {
 	defaultValue := &v1alpha1.Experiment{}
 	assert.Equal(t, DefaultProgressDeadlineSeconds, GetExperimentProgressDeadlineSecondsOrDefault(defaultValue))
 }
+
+func TestGetConsecutiveErrorLimitOrDefault(t *testing.T) {
+	errorLimit := (*int32)(nil)
+	metricNonDefaultValue := v1alpha1.Metric{}
+	assert.Equal(t, errorLimit, metricNonDefaultValue.ConsecutiveErrorLimit)
```

Comment:
I don't think we need to have this assertion since the `metricNonDefaultValue` declaration above implies that the `ConsecutiveErrorLimit` is nil